File: tooling/sdkgen/test/typescript/openws-sdkgen-typescript-node/src/core/models/server/join-room-payload.ts
export interface JoinRoomPayloadInit {
    userId: string
    roomId: string
}

export class JoinRoomPayload implements JoinRoomPayloadInit {
    readonly userId: string
    readonly roomId: string

    constructor({ userId, roomId }: JoinRoomPayloadInit = {} as JoinRoomPayloadInit) {
        this.userId = userId
        this.roomId = roomId
    }
}

File: tooling/sdkgen/test/dotnet/UnityTestbed/Assets/chat/core/Runtime/Example.Chat.Core.Sdk/Models/Server/JoinRoomPayload.cs
using Newtonsoft.Json;
using System.Collections.Generic;

namespace Example.Chat.Core.Models.Server
{
    public partial class JoinRoomPayload
    {
        [JsonProperty("userId")]
        public string UserId;

        [JsonProperty("roomId")]
        public string RoomId;

    }
}